integration_tests/e2e/add-steps.cy.ts

```diff
@@ -0,0 +1,174 @@
+import AddSteps from '../pages/add-steps'
+import { PlanType } from '../../server/@types/PlanType'
+import DataGenerator from '../support/DataGenerator'
+
+describe('Add Steps', () => {
+  const addStep = new AddSteps()
+
+  const selectStepDescriptionByIndex = (index: number) => {
+    return cy.get(`table.goal-summary-card__steps .govuk-table__body > :nth-child(${index}) > :nth-child(1)`)
+  }
+
+  const selectStepActorByIndex = (index: number) => {
+    return cy.get(`table.goal-summary-card__steps .govuk-table__body > :nth-child(${index}) > :nth-child(2)`)
+  }
+
+  beforeEach(() => {
+    cy.createSentencePlan().then(planDetails => {
+      cy.wrap(planDetails).as('plan')
+      cy.openSentencePlan(planDetails.oasysAssessmentPk)
+    })
+  })
+
+  describe('Adding steps', () => {
+    beforeEach(() => {
+      cy.get<{ plan: PlanType }>('@plan').then(({ plan }) => {
+        const goalData = DataGenerator.generateGoal()
+        cy.addGoalToPlan(plan.uuid, goalData).then(goal => {
+          cy.visit(`/goal/${goal.uuid}/add-steps`)
+        })
+      })
+    })
+
+    it('Add one step', () => {
+      cy.url().should('include', '/add-steps')
+      const stepDescription = 'This is the first step'
+      const stepActor = 'Sam'
+      addStep.addStepAutocompleteText(1, stepDescription)
+      addStep.selectStepActor(1, stepActor)
```

Comment:
Any reason you didn't use the DataGenerator.generateStep() for generating some of this info? could have done
```
const step = DataGenerator.generateStep()

addStep.addStepAutocompleteText(1, step.description)
addStep.selectStepActor(1, step.actor)
```

No real difference in the end result, but it just means that, like in later steps, if you're creating multiple steps you don't need to make multiple step related variables

Reply:
because I forgot it existed. I will do that, much better.

Reply:
Addressed in https://github.com/ministryofjustice/hmpps-sentence-plan-ui/pull/124/commits/3e866574dd4374e959a1bf04f0401f6c8616474a